### PR TITLE
4 New KML examples

### PR DESCRIPTION
**Pull request by maikuru**

### CODE REVIEW

File: examples/kmlFetchGood.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--
+See https://github.com/AnalyticalGraphicsInc/cesium-google-earth-examples/blob/master/LICENSE.md
+
+Fenway Model was used from the publicly available Model on Sketchup
+https://3dwarehouse.sketchup.com/model.html?id=21eee70f917ce34af7a0c817ab72c49b
```
Comment (pjcozzi) [inline examples/kmlFetchGood.html]: Did you check the [TOS](https://3dwarehouse.sketchup.com/tos.html)?  Are you sure we can use this?  In the past, we were not able to use models when this was Google's warehouse.

File: examples/kmlFetchInteractive.html [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+                // failure is often a result of CORS Security Policies
+                // See https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS
+            });
+    }
+
+    function relevantSource() {
+          var kmlInput = document.getElementById('kmlUrl');
+          var errorMsg = document.getElementById('errorMessage');
+//
+//        function onLoadKml() {
```
Comment (pjcozzi) [inline examples/kmlFetchInteractive.html]: Is this supposed to be here and commented out?

File: examples/kmlParse.html [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+        <div class="panel-heading">Relevant Cesium Source</div>
+        <div class="panel-body">
+            <textarea id="cesiumSource"></textarea>
+        </div>
+    </div>
+    <div class="panel panel-primary" id="gePanel">
+        <div class="panel-heading">Relevant Google Earth Source (<a target="_blank" href="https://code.google.com/p/earth-api-samples/source/browse/trunk/examples/hittest.html">link</a>)</div>
+        <div class="panel-body">
+                <textarea id="gesource">
+                    // Sample KML taken from
```
Comment (pjcozzi) [inline examples/kmlParse.html]: Was this KML already in the original code?  If so, it should be covered by the Apache 2.0 license and OK to use.  If it was just on their website and we added it to the code, we'll need to find or create another KML example.

Comment (maikuru) [inline examples/kmlParse.html]: Yes the KML below is from the original source.  I should be covered by Apache 2 license.  The KML in my implementations are custom.

File: examples/kmlFetchInteractive.html (hunk 2 above)
Comment (maikuru) [inline examples/kmlFetchInteractive.html]: Yes, the viewSource method renders it as though it's not commented out.

File: examples/kmlFetchGood.html (hunk 1 above)
Comment (maikuru) [inline examples/kmlFetchGood.html]: I had read the TOS, but did not see any exclusions.  It's moot anyways as I didn't end up using the Fenway model as we don't support loading Models from sketchup via a KML export.

**Comment by pjcozzi** [comment]
Looks good, thanks.

**Comment by pjcozzi** [comment]
Also, in general, when we merge to `master` in this repo, please merge it to `gh-pages` to deploy.

**Pull request merged**